
import sys

# 指定非0异常退出
# if __name__ == "__main__":
#     try:
#         print("x-------")
#         a = 1/0
#         print("y-------")
#     except Exception as e:
#         print(repr(e))
#         # sys.exit(1)
#     print("over")

# 非0退出
if __name__ == "__main__":
    a = 1/0    